exercises/prime-factors/README.md

```diff
@@ -0,0 +1,28 @@
+Compute the prime factors of a given natural number.
```

Comment:
The README should be generated using `configlet`, as outlined in the Contributors Guide, full details are here: https://github.com/exercism/docs/blob/master/language-tracks/exercises/anatomy/readmes.md